.circleci/config.yml

```diff
@@ -0,0 +1,103 @@
+version: 2
+
+jobs:
+  build_36:
+    machine: true
+    environment:
+      PYTHON: "3.6"
+      ENV_NAME: "py36-sparse-test"
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - miniconda-v1-{{ checksum "ci/environment-3.6.yml" }}
+      - run:
+          name: install miniconda
+          command: |
+              if [ ! -d "/home/circleci/miniconda" ]; then
+                wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
+                bash miniconda.sh -b -p $HOME/miniconda
+                export PATH="$HOME/miniconda/bin:$PATH"
+                conda config --set always_yes yes --set changeps1 no
+              fi
+              sudo chown -R $USER.$USER $HOME
+      - run:
+          name: configure conda
+          command: |
+              export PATH="$HOME/miniconda/bin:$PATH"
+              if [ ! -d "/home/circleci/miniconda/envs/py36-sparse-test" ]; then
+                conda update -q conda
+                conda env create -f ci/environment-${PYTHON}.yml --name=${ENV_NAME}
+                source activate ${ENV_NAME}
+              fi
+              conda env list
+              conda list ${ENV_NAME}
+      - save_cache:
+          key: miniconda-v1-{{ checksum "ci/environment-3.6.yml" }}
+          paths:
+            - "/home/circleci/miniconda"
+      - run:
+          command: |
+            export PATH="$HOME/miniconda/bin:$PATH"
+            source activate ${ENV_NAME}
+            pip install -e .[tests]
+      - run:
+          command: |
+            /home/circleci/miniconda/envs/py36-sparse-test/bin/pytest --flake8
```

Comment:
Can you cd into the project directory and run `pytest`? It'll automatically pick up `pytest.ini` that way. You also won't need `--flake8`, it's already in `pytest.ini`.

This also makes the coverage not generate.